app/components/table-message/no-result.hbs

```diff
@@ -0,0 +1,11 @@
+{{!-- No Result Message --}}
+<TableMessage>
+    <p class="au-u-margin-bottom-large no-result">No result Er werden geen zoekresultaten gevonden. Kijk na op spelfouten, probeer een andere zoekopdracht of voeg een nieuwe persoon toe!</p>
+    <AuLink
+        @route="people.new"
+        @query={{hash redirectUrl=@personCreationRedirectUrl}}
```

Comment:
The redirection doesn't work as expected here, I think it's because `@personCreationRedirectUrl` is not passed down to the component ?

Reply:
@claire-lovisa I checked it, and it redirects to http://localhost:4200/personen/nieuw

Reply:
I mean now, when do create the new person in the flow to create a new position, you are then redirected to `/personen/:id/contactgegevens` instead of the position creation page and that is fixed by calling the component like ```<TableMessage::NoResult @personCreationRedirectUrl={{@personCreationRedirectUrl}}/>```